time stamp
eac_pkg/ObjectInfo[] data